app/uk/gov/hmrc/apiplatformoutboundsoap/models/JsonFormats.scala

```diff
@@ -16,65 +16,71 @@
 
 package uk.gov.hmrc.apiplatformoutboundsoap.models
 
+import java.time.format.DateTimeFormatterBuilder
+import java.time.{Instant, ZoneId}
+import java.util.UUID
+
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
-import uk.gov.hmrc.apiplatformoutboundsoap.config.AppConfig
 
-import java.time.Instant
-import java.util.UUID
+import uk.gov.hmrc.apiplatformoutboundsoap.config.AppConfig
 
 object JsonFormats {
 
   import uk.gov.hmrc.apiplatformoutboundsoap.GlobalContext.injector
 
-  val appConfig: AppConfig = injector.instanceOf[AppConfig]
+  val appConfig: AppConfig                                    = injector.instanceOf[AppConfig]
   implicit val privateHeaderFormatter: OFormat[PrivateHeader] = Json.format[PrivateHeader]
 
   implicit val instantFormat: Format[Instant] = Format(Reads.DefaultInstantReads, Writes.InstantEpochMilliWrites)
 
-  val addressingReads: Reads[Addressing] = (
+  val addressingReads: Reads[Addressing]                                          = (
     (JsPath \ "from").read[String].orElse(Reads.pure(appConfig.addressingFrom)) and
       (JsPath \ "to").read[String] and
       (JsPath \ "replyTo").read[String].orElse(Reads.pure(appConfig.addressingReplyTo)) and
       (JsPath \ "faultTo").read[String].orElse(Reads.pure(appConfig.addressingFaultTo)) and
       (JsPath \ "messageId").read[String] and
       (JsPath \ "relatesTo").readNullable[String]
-    )(Addressing.apply _)
-  implicit val addressingFormatter: OFormat[Addressing] = OFormat(addressingReads, Json.writes[Addressing])
-  val statusWrites: Writes[SoapMessageStatus] = (
+  )(Addressing.apply _)
+  implicit val addressingFormatter: OFormat[Addressing]                           = OFormat(addressingReads, Json.writes[Addressing])
+
+  val statusWrites: Writes[SoapMessageStatus]                                     = (
     (JsPath \ "globalId").write[UUID] and
       (JsPath \ "messageId").write[String] and
       (JsPath \ "status").write[StatusType] and
       (JsPath \ "ccnHttpStatus").write[Int] and
-      (JsPath \ "sentDateTime").writeOptionWithNull[Instant](Writes.DefaultInstantWrites) and
+      (JsPath \ "sentDateTime").writeOptionWithNull[Instant](Writes.temporalWrites(new DateTimeFormatterBuilder()
```

Comment:
You might want to pull this out as a val unless you're sure it's not added as a function type